library/Icinga/Application/Test.php

```diff
@@ -74,7 +74,7 @@ public function setupAutoloader()
         }
 
         // Conflicts with `Tests\Icinga\Module\...\Lib`. But it seems it's not needed anyway...
-        //$this->getLoader()->registerNamespace('Tests', $this->getBaseDir('test/php/library'));
+        $this->getLoader()->registerNamespace('Tests\\Icinga\\Lib', $this->getBaseDir('test/php/Lib'));
```

Comment:
I think you should leave the commented out code and comment above as is, but add your code separately instead. After all, your change has nothing to do with the existing commented out code and comment.